### PR TITLE
added better navigating by tags. 

### PR DESCRIPTION
**Pull request by tam203**
Refine searches by clicking a tag to add it to the search.
Click an existing tag to remove it from the search.

### CODE REVIEW

**Comment by tam203** [comment]
related to #19

File: src/server.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ app.use('/img/', proxy('moc-vr.s3-eu-west-1.amazonaws.com/', {
 
 //view endpoints
 app.get("/", function (req, res) {
+    console.log('in /');
```
Comment (tpowellmeto) [inline src/server.js]: no logging!

File: src/mocvr-data-service.js [hunk 2]
```diff
@@ -170,17 +170,23 @@ module.exports = {
         });
     },
 
-    findByTag: function(tag) {
+    findByTags: function(tags) {
         return new Promise(function (resolve, reject) {
+            let expressionAttributeValues = {};
```
Comment (tpowellmeto) [inline src/mocvr-data-service.js]: swap all 'let' for 'var' - you're too modern!

**Comment by tam203** [comment]
Got spots. @tpowellmeto Now done.

**Pull request merged**